kubernetes/samples/scripts/terraform/provider.tf

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018, 2021, Oracle and/or its affiliates.
+ * Copyright (c) 2020, 2022, Oracle and/or its affiliates.
```

Comment:
Why did you change the original date?

Reply:
I think it was copied from somewhere else, did we have operator in 2018? should I leave it as 2018?

Reply:
Operator 1.0 was released in 2018. If "2020" is the correct begin date then go ahead and fix it. I'm just reviewing the changes that are presented here :).